### ISSUE TITLE
Idle animation not looping

### ISSUE THREAD

**Issue by NicolasBrondin** [pmndrs/ecctrl#67]
When setting my `idle` animation, I was expecting it to loop like the `walk`, but instead it gets played only once, after the character stays static.

Thank you for your amazing job 🙏

**Comment by ErdongChen-Andrew** [comment]
Thanks @NicolasBrondin ! Did you set up the animation name correctly?
<img width="779" alt="截屏2024-05-27 17 19 16" src="https://github.com/pmndrs/ecctrl/assets/84251107/3a89cc9e-1c35-403d-b149-12d5ef4dca17">

**Comment by NicolasBrondin** [comment]
Yes absolutely ! Otherwise it would not trigger at all I guess! Right now it's played once when the player loads, then it stops (but it does not come back to the T shape, it stays in the same position than the last frame of the idle animation)

**Comment by ErdongChen-Andrew** [comment]
Did you use the animation multiple times? For example, did you put "idle" in "fall"? I can't help much without seeing any code. Maybe you can check the example app setups here: https://github.com/pmndrs/ecctrl/blob/main/example/Experience.tsx.

**Comment by NicolasBrondin** [comment]
@ErdongChen-Andrew Yes I did! I will try with different animations to see if it fixes the issue 🙏

**Comment by MULUALEM-TEKLE** [comment]
making sure no animation action is reused is a must, it fixed a similar issue in my case, thanks @ErdongChen-Andrew
